src/node_file.h

```diff
@@ -9,31 +9,38 @@
 namespace node {
 
 using v8::Context;
+using v8::FunctionCallbackInfo;
 using v8::HandleScope;
 using v8::Local;
+using v8::MaybeLocal;
 using v8::Object;
+using v8::Persistent;
+using v8::Promise;
+using v8::PropertyCallbackInfo;
+using v8::String;
 using v8::Undefined;
 using v8::Value;
 
 namespace fs {
 
-class FSReqWrap: public ReqWrap<uv_fs_t> {
+enum Ownership { COPY, MOVE };
```

Comment:
Can you make this `enum class`? I think that would be a bit more readable since `COPY` and `MOVE` are pretty short identifiers and it’s not obvious what kind of thing they refer to…

Reply:
Also good call